src/components/Book.jsx

```diff
@@ -1,34 +1,48 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { AiOutlineLoading3Quarters } from 'react-icons/ai';
 import BookstoreAPI from '../services/BookstoreAPI';
 
 const Book = ({
-  id, category, title, author, currentChapter,
+  id, category, title, author, currentChapter, totalChapters,
 }) => {
   const dispatch = useDispatch();
   const handleClick = () => {
     dispatch(BookstoreAPI.deleteBookFromAPI(id));
     dispatch(BookstoreAPI.getAllBooksFromAPI());
   };
+  const progressPercentage = Math.round((currentChapter / totalChapters) * 100);
   return (
-    <div className="book-container">
-      <div className="book-info">
+    <div className="book-container d-flex flex-row">
+      <div className="d-flex flex-column book-info">
         <h3 className="book-category">{category}</h3>
         <h2 className="book-title">
           {title}
         </h2>
         <h4 className="book-author">{author}</h4>
-        <div className="options">
-          <button type="button">Comments</button>
-          <button type="button" onClick={handleClick}>Remove</button>
-          <button type="button">Edit</button>
+        <div className="d-flex flex-row">
+          <button type="button" className="btn-link">Comments</button>
+          <button type="button" onClick={handleClick} className="btn-link">Remove</button>
+          <button type="button" className="btn-link">Edit</button>
         </div>
```

Comment:
- [ ] You have done a good job so far. However, these elements have a right border in the Zeplin design, please let's add it to make sure our design matches that of Zeplin.

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/nSuTOLw.png) | ![](https://i.imgur.com/UYndIsi.png)